Makefile

```
@@ -177,12 +177,28 @@ run-mock-management-grpc-server: ## Run mock management plane gRPC server
 	@echo "🖲️ Running mock management plane gRPC server"
 	$(GORUN) test/mock/grpc/cmd/main.go -configDirectory=$(MOCK_MANAGEMENT_PLANE_CONFIG_DIRECTORY) -logLevel=$(MOCK_MANAGEMENT_PLANE_LOG_LEVEL) -grpcAddress=$(MOCK_MANAGEMENT_PLANE_GRPC_ADDRESS) -apiAddress=$(MOCK_MANAGEMENT_PLANE_API_ADDRESS)
 
-generate: ## Generate proto files and server and client stubs from OpenAPI specifications
+build-test-plus-image: local-deb-package
+	$(CONTAINER_BUILDENV) $(CONTAINER_CLITOOL) build -t nginx_plus_${IMAGE_TAG} . \
+		--no-cache -f ./scripts/docker/nginx-plus/deb/Dockerfile \
+		--secret id=nginx-crt,src=${CERTS_DIR}/nginx-repo.crt \
+		--secret id=nginx-key,src=${CERTS_DIR}/nginx-repo.key \
+		--build-arg PACKAGE_NAME=${PACKAGE_NAME} \
+		--build-arg PACKAGES_REPO=${OSS_PACKAGES_REPO} \
+		--build-arg BASE_IMAGE=${BASE_IMAGE} \
+		--build-arg ENTRY_POINT=./scripts/docker/entrypoint.sh
```

nit: could we use parentheses consistently? Becomes very messy over time if we just randomly use both.
```suggestion
	$(CONTAINER_BUILDENV) $(CONTAINER_CLITOOL) build -t nginx_plus_$(IMAGE_TAG) . \
		--no-cache -f ./scripts/docker/nginx-plus/deb/Dockerfile \
		--secret id=nginx-crt,src=$(CERTS_DIR)/nginx-repo.crt \
		--secret id=nginx-key,src=$(CERTS_DIR)/nginx-repo.key \
		--build-arg PACKAGE_NAME=$(PACKAGE_NAME) \
		--build-arg PACKAGES_REPO=$(OSS_PACKAGES_REPO) \
		--build-arg BASE_IMAGE=$(BASE_IMAGE) \
		--build-arg ENTRY_POINT=./scripts/docker/entrypoint.sh
```